packages/components/Menu/src/MenuList/useMenuList.ts

```diff
@@ -162,9 +162,21 @@ export const useMenuList = (_props: MenuListProps): MenuListState => {
     };
   });
 
+  // focus management
+  const focusZoneRef = React.useRef<View>();
+  const setFocusZoneFocus = () => {
+    focusZoneRef?.current?.focus();
+  };
+
+  React.useEffect(() => {
+    setFocusZoneFocus();
```

Comment:
Why not just call `focusZoneRef?.current?.focus();` directly here?

Reply:
I think it was a bit of code sharing done with the onMouseLeave call below. This code was moved from MenuList.tsx to consolidate the hooks into one place. I think it's fine as is, since we want the same thing to happen on mount as onMouseLeave, it seems.